src/vs/workbench/contrib/terminal/common/terminal.ts

```diff
@@ -30,6 +30,9 @@ export const KEYBINDING_CONTEXT_TERMINAL_IS_TABS_NARROW_FOCUS = new RawContextKe
 /** A context key that is set when the integrated terminal tabs widget has focus. */
 export const KEYBINDING_CONTEXT_TERMINAL_TABS_FOCUS = new RawContextKey<boolean>('terminalTabsFocus', false, nls.localize('terminalTabsFocusContextKey', "Whether the terminal tabs widget is focused"));
 
+/** A context key that is set when the integrated terminal tabs widget has the mouse focus. */
+export const KEYBINDING_CONTEXT_TERMINAL_TABS_MOUSE = new RawContextKey<boolean>('terminalTabsMouse', false, nls.localize('terminalTabsMouseContextKey', "Whether the mouse is within the terminal tabs widget"));
```

Comment:
I think we should avoid the description here, if we include it then it shows up on the website which means people might try to use it (which is undesirable as it could change in the future).